exporting/prometheus/remote_write/remote_write.c

```diff
@@ -82,6 +82,18 @@ int process_prometheus_remote_write_response(BUFFER *buffer, struct instance *in
         return exporting_discard_response(buffer, instance);
 }
 
+/**
+ * Clean up remote write instance
+ *
+ * @param instance an instance data structure.
+ */
+void clean_prometheus_remote_write_instance(struct instance *instance)
+{
+    buffer_free(instance->buffer);
+
+    freez(instance->connector_specific_data);
```

Comment:
```
struct prometheus_remote_write_specific_data {
    void *write_request;
};
```
```
struct prometheus_remote_write_specific_config {
    char *remote_write_path;
};
```
We need to free them. As of `write_request`, it might be freed by `protocol_buffers_shutdown()` but we need to check it.

Reply:
When I tried to clean it alone, the library reported Fatal, but when I called only `protocol_buffers_shutdown()`, it was closed without error. Reading the [official documentation](https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.common), I think we do not need to change this.

Reply:
We still have to clean `connector_specific_data ` (without `write_request`) and `remote_write_path`. We should call it from the simple connector worker before cleaning `connector_specific_config`. Don't forget #ifdef.